### PR TITLE
add net.sourceforge.audme.audmes

### PR DESCRIPTION
**Pull request by swwa**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by swwa** [comment]
This pull request replaces https://github.com/flathub/flathub/pull/4152. Domain changed to Sourceforge.

**Comment by swwa** [comment]
bot, build  net.sourceforge.audmes.audmes

**Comment by swwa** [comment]
bot, build net.sourceforge.audmes.audmes

File: net.sourceforge.audmes.audmes.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        },
+        {
+            "name": "audmes",
+            "buildsystem": "cmake-ninja",
+            "builddir": "true",
+            "sources": [
+                    {
+                    "type": "git",
+                    "url": "https://git.code.sf.net/p/audmes/git",
+		    "branch": "master",
```
Comment (hfiguiere) [inline net.sourceforge.audmes.audmes.json:57]:
```suggestion
```

never a branch.

Comment (swwa) [reply net.sourceforge.audmes.audmes.json]: @hfiguiere  removed

File: net.sourceforge.audmes.audmes.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+              "--with-libiconv=/usr"
+          ],
+          "build-options" : {
+              "cxxflags":"-std=c++11"
+          },
+          "sources": [ 
+            {  
+                "type": "archive", 
+                "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.5.1/wxWidgets-3.0.5.1.tar.bz2",
+                "sha1":"406ac736f61d88a3a866aa501e01e408a642c6e7"
```
Comment (hfiguiere) [inline net.sourceforge.audmes.audmes.json:45]: please don't use sha1. it is deprecated. we usually use sha256.

Comment (swwa) [reply net.sourceforge.audmes.audmes.json]: checksum changed to sha256

**Comment by swwa** [comment]
bot, build net.sourceforge.audmes.audmes

**Comment by swwa** [comment]
@hfiguiere All changes made.

**Pull request closed without merging**